core/output/facebook.js

```diff
@@ -71,23 +115,54 @@ exports.start = function(callback) {
 					endTyping = null;
 				}
 				api.setTitle(title, thread);
+			},
```

Comment:
I also noticed 
```
if (endTyping != null) {
                    endTyping();
                    endTyping = null;
}
```
can be extracted into a separate method as it's being repeated 5 times